packages/client/src/app/views/LoginView.vue

```diff
@@ -17,26 +17,28 @@ function login() {
     <section class="row h-100 d-flex align-items-center">
       <div class="col-xl-6 col-sm-12 mb-5">
         <h1 class="mb-5">Login</h1>
-        <form @submit.prevent="login">
+        <FormKit type="form" @submit="login" :actions="false" :incomplete-message="false">
           <InputLabel
             v-model:modelValue="user.username"
-            :label="'Gebruikersnaam'"
-            :placeholder="'Gebruikersnaam'"
+            label="Gebruikersnaam"
+            validation="required"
+            placeholder="Gebruikersnaam"
```

Comment:
Just something to think about: According to [this article](https://www.nngroup.com/articles/form-design-placeholders/) an placeholder should be used to provide extra context, a hint or an example of the required format for the input field, rather than just repeating the label

Reply:
Good point. I will change it